package.json

```diff
@@ -131,7 +132,12 @@
         "URL_SCHEME": "superhero",
         "DEEPLINK_HOST": "wallet.superhero.com"
       },
-      "cordova-plugin-qrscanner": {}
+      "cordova-plugin-qrscanner": {},
+      "cordova-plugin-openwith": {
+        "IOS_URL_SCHEME": "hf938wofijfjhsdlsf23",
```

Comment:
Can you let us know what this `IOS_URL_SCHEME` is about?

Reply:
It's supposed to be random: https://github.com/aeternity/cordova-plugin-openwith#installation

Reply:
Cool. Thanks!